packages/theme/src/pageTheme.ts

```diff
@@ -63,4 +63,5 @@ export const pageTheme: Record<string, PageTheme> = {
   library: genPageTheme(colorVariants.rubyRed, shapes.wave),
   other: genPageTheme(colorVariants.darkGrey, shapes.wave),
   app: genPageTheme(colorVariants.toastyOrange, shapes.wave),
+  apis: genPageTheme(colorVariants.eveningSea, shapes.wave2),
```

Comment:
Do we really want to go with that color? 
![image](https://user-images.githubusercontent.com/648527/114431457-cc163b80-9bbf-11eb-9dc9-6e6296e9bc4d.png)

It's hard to read

Reply:
Yep I think the variant needs updating. Need @katz95 input here.

Reply:
@benjdlambert I created a PR to rollback the color #5320